dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/backend/Backend.scala

```diff
@@ -0,0 +1,11 @@
+package me.shadaj.scalapy.tensorflow.api.keras.backend
+import me.shadaj.scalapy.tensorflow.keras.backend.{Backend => PyBackend}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.{PythonModule, PythonType}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption._
+import scala.language.implicitConversions
+import me.shadaj.scalapy.py.PyFunction
+
+class Backend private[api] (val underlying: PyBackend) extends PythonType[PyBackend] with PythonModule {
```

Comment:
Hmm this `PythonType[PyBackend] with PythonModule` piece is somewhat confusing (as in, suggests that this class corresponds both to a Python type AND a Python module) :thinking: ... is `PythonModule[Backend]` a viable option?

Reply:
oke